evolution/genome.py

```diff
@@ -34,6 +34,23 @@ def __init__(self, connections, input_size, output_size):
         self.output_node_ids = []
         self.fitness = None
 
+        # copy another genome
+        if genome_to_clone is not None:
+            self.input_size = genome_to_clone.input_size
+            self.output_size = genome_to_clone.output_size
+
+            for node_gene in genome_to_clone.get_nodes():
+                self.node_genes[node_gene.node_id] = NodeGene(node_gene.node_id, node_gene.node_type)
+
+            for connection in genome_to_clone.connection_genes.values():
+                self.connection_genes[(connection.source_node.node_id, connection.destination_node.node_id)] = \
```

Comment:
trzeba to troszkę skrócić, przypisz `source_node.node_id` itd. do pomocnicznych zmiennych